redaxo/src/core/pages/system.settings.php

```diff
@@ -115,6 +115,10 @@
                         <p>' . rex_i18n::msg('delete_cache_description') . '</p>
                         <p><a class="btn btn-delete" href="' . rex_url::currentBackendPage(['func' => 'generate'] + $csrfToken->getUrlParams()) . '">' . rex_i18n::msg('delete_cache') . '</a></p>
 
+                        <h3>' . rex_i18n::msg('safemode') . '</h3>
+                        <p>' . rex_i18n::msg('safemode_text') . '</p>
+                        <p><a class="btn-setup" href="' . rex_url::currentBackendPage(['safemode' => 'true'] + $csrfToken->getUrlParams()) . '" data-pjax="false">' . rex_i18n::msg('safemode_activate') . '</a></p>
```

Comment:
hier müsste `class="btn btn-setup"` hin. weiß nicht, wie ich das hier in den PR noch reinbekomme. Neuer Patch? @gharlan

Reply:
Inwiefern hat der Button etwas vom Setup?

Reply:
> weiß nicht, wie ich das hier in den PR noch reinbekomme. Neuer Patch?

Da hier schon gemerged ist, musst du einen neuen PR machen.

Reply:
@tbaddade ich wusste nicht, wie der Button dieselbe Farbe erhält, wie der Setup-Button. Ok, ich mache dann einen neuen PR. Sorry, wollte jetzt nicht mehr Aufwand produzieren.

Reply:
Fände es gut, wenn der sogar auf das Orange vom Safemode zurückgreift..